library/core/src/array/mod.rs

```diff
@@ -364,3 +364,61 @@ macro_rules! array_impl_default {
 }
 
 array_impl_default! {32, T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T}
+
+#[cfg(not(bootstrap))]
+#[lang = "array"]
+impl<T, const N: usize> [T; N] {
+    /// Returns an array of the same size as `self`, with `f` applied to each element.
```

Comment:
Should we change this to
```suggestion
    /// Apply function `f` to each element in the array.
```
I don't know if mentioning that the same size is useful since that is in the type signature and similar to `Iterator::map` but I think we should clarify that `f` is a function.

Reply:
ah I can clarify that `f` is a function, but I think it is useful to specify that it's another array returned